assembl/templates/index_react.jinja2

```diff
@@ -50,5 +50,18 @@
                 <div id="under-test" class="hidden">V2</div>
             {%- endif %}
         {% endblock %}
+
+        {% if inject_bugherd %}
+            <!-- Bugherd injection -->
+            <script type='text/javascript'>
+            (function (d, t) {
+            var bh = d.createElement(t), s = d.getElementsByTagName(t)[0];
```

Comment:
You should get the last element and append it there. You'll be delaying the loading of `bundle.js` with this code. Doesn't make much difference in testing env, but just letting you know, as in the template, the bundle.js is the last `script` tag that is declared in the HTML.